server/manager/src/main/java/org/apache/accumulo/manager/recovery/RecoveryManager.java

```diff
@@ -156,72 +156,68 @@ private boolean exists(final Path path) throws IOException {
     }
   }
 
-  public boolean recoverLogs(KeyExtent extent, Collection<Collection<String>> walogs)
-      throws IOException {
+  public boolean recoverLogs(KeyExtent extent, Collection<LogEntry> walogs) throws IOException {
```

Comment:
Protip: this diff looks bigger than it is in GitHub. Best to ignore whitespace / indentation while comparing for a more friendly view.